guides/source/routing.md

```diff
@@ -1278,8 +1314,8 @@ edit_video GET  /videos/:identifier/edit(.:format) videos#edit
 Video.find_by(identifier: params[:identifier])
 ```
 
-You can override `ActiveRecord::Base#to_param` of the associated model to construct
-a URL:
+You can override `ActiveRecord::Base#to_param` of the associated model to
+construct a URL:
```

Comment:
I'm not certain that it's obvious to a beginner what we're doing in this example, I can't think of a way offhand but do you think there's any more detail we can add to further explain what we're doing here and what it's helping us achieve? In addition, is there a link to the API for `ActiveRecord::Base#to_param` that we could link to?

Reply:
Added a bit more to clarify and link to `to_param`